app/uk/gov/hmrc/apiplatform/modules/approvals/domain/models/ResponsibleIndividualVerificationState.scala

```diff
@@ -21,7 +21,7 @@ import uk.gov.hmrc.thirdpartyapplication.domain.utils.EnumJson
 object ResponsibleIndividualVerificationState extends Enumeration {
   type ResponsibleIndividualVerificationState = Value
 
-  val INITIAL, REMINDERS_SENT = Value
+  val INITIAL, REMINDERS_SENT, ADMIN_REQUESTED_CHANGE = Value
```

Comment:
We will probably want to remove this new state if we are going for the new 'type' field.  But I guess we can always remove it.